### PR TITLE
Extended timeout

### LINKED ISSUE #780: SendTip: fail to reload data after tip was send
Stuck like this after tip was send
SendTip
```
util.js:24 Uncaught (in promise) Error: Request is cancelled by timeout
vue.runtime.esm.js:1888 Error: Request is cancelled by timeout
```

### PR DESCRIPTION
**Pull request by AtanasKrondev**
Fixes #780 

Extended requests timeout and after sending tip the reloading is delayed, to ensure enough time the back-end to process it.

### CODE REVIEW

File: src/components/layout/sendTip/SendTip.vue [hunk 1]
```diff
@@ -111,12 +111,12 @@ export default {
         .then(async () => {
           await Backend.cacheInvalidateTips().catch(console.error);
           this.clearTipForm();
-          EventBus.$emit('reloadData');
           this.$store.dispatch('modals/open', {
             name: 'success',
             title: this.$t('components.layout.SendTip.SuccessHeader'),
             body: this.$t('components.layout.SendTip.SuccessText'),
           });
+          setTimeout(() => EventBus.$emit('reloadData'), 5000);
```
Comment (mradkov) [inline src/components/layout/sendTip/SendTip.vue:119]: Why is this timeout needed here?

Comment (AtanasKrondev) [reply src/components/layout/sendTip/SendTip.vue:119]: The backend itself take about 5 secs to process the tip, so it was suggested not to reload immediately after sending the tip, but delay a little bit.

Comment (mradkov) [reply src/components/layout/sendTip/SendTip.vue:119]: Is delay of `5000` ms enough?

Comment (AtanasKrondev) [reply src/components/layout/sendTip/SendTip.vue:119]: with 5000 ms delay for sending the reload request plus 12 000 ms timeout before throwing the error even for slow connections should be fine.

**Review by mradkov — APPROVED** [review]

**Pull request merged**